gspread/client.py

```diff
@@ -193,7 +194,13 @@ def create(self, title, folder_id=None):
         if folder_id is not None:
             payload['parents'] = [folder_id]
 
-        r = self.request('post', DRIVE_FILES_API_V3_URL, json=payload)
+        params = None
+        if supports_all_drives:
+            params = {
+                'supportsAllDrives': True
+            }
```

Comment:
It works if we set the parameter to `True` even if the user want to save the file in its own drive.

I recommend here, to always create variable `params` and set the value of `'supportsAllDrives'` to `supports_all_drives`